lib/python/rose/bush.py

```diff
@@ -442,7 +443,8 @@ def summary(self, user, form=None):
         return self.suites(user, form)
 
     @cherrypy.expose
-    def view(self, user, suite, path, path_in_tar=None, mode=None):
+    def view(self, user, suite, path, path_in_tar=None, mode=None,
+             search_string=None, search_mode='TEXT'):
```

Comment:
Please follow the general style of this source file and use mainly double quotes for literal strings.